### PR TITLE
seed FAQ document with content from the Slack AMA session

### LINKED ISSUE #21: Seed FAQ section or document
We had a Slack AMA session ([link to beginning of it](https://kubernetes.slack.com/archives/CBT6N1ASG/p1574693499371400)) and it'd be great to massage some of the questions and replies into an FAQ doc or a section on the main page.

### PR DESCRIPTION
**Pull request by dholbach**
closes #21

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+- Syncing will be more efficient. Instead of polling, flux will use Kubernetes Informers to get information from the cluster.
+- Users will see a great (if not huge) reduction in K8S API calls and etcd traffic
+We can increase the syncing frequency.
+- Advanced syncing features such as pre-post sync hooks and sync waves
+- Overall performance and efficiency improvements (registry scanning excluded) are the major gains for Flux users.
+- Given the concept GitOps is quite young, and all people involved were involved early on (or present during the birth), I think people get an even more experienced team working on core features.
+
+----
+
+**Q**: What can current Argo users look forward to from this collaboration with Flux?
```
Comment (jaypipes) [inline docs/faq.md]: s/Argo/Argo CD/

File: docs/faq.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+**A**: Our primary motives for coming together to do this are:
+
+- ArgoCD and Flux are two of the main GitOps projects, solving very similar problems, having very similar views on implementing GitOps.
+- We want to offer a shared vision for GitOps and the best possible GitOps experience for everyone.
+- We hope to bring a bigger community together than we can on our own.
+- We want to learn from each other's approaches and offer the best in breed GitOps solution out there.
+
+----
+
+**Q**: What can current Flux users look forward to from this collaboration with Argo?
```
Comment (jaypipes) [inline docs/faq.md]: s/Flux/Flux CD/
s/Argo/Argo CD/

File: docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+- Pratik Wadher on the [Intuit blog](https://www.intuit.com/blog/technology/introducing-argo-flux/)
+- Tamao Nakahara on the [Weaveworks blog](https://www.weave.works/blog/argo-flux-join-forces)
+
+----
+
+**Q**: What are you hoping to get out of this collaboration?
+
+**A**: Our primary motives for coming together to do this are:
+
+- ArgoCD and Flux are two of the main GitOps projects, solving very similar problems, having very similar views on implementing GitOps.
```
Comment (jaypipes) [inline docs/faq.md]: s/Flux/Flux CD/

File: docs/faq.md [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+- We hope to bring a bigger community together than we can on our own.
+- We want to learn from each other's approaches and offer the best in breed GitOps solution out there.
+
+----
+
+**Q**: What can current Flux users look forward to from this collaboration with Argo?
+
+**A**: Here are a few of our favourites:
+
+- Syncing will be more efficient. Instead of polling, flux will use Kubernetes Informers to get information from the cluster.
```
Comment (jaypipes) [inline docs/faq.md]: s/flux/Flux CD/

File: docs/faq.md [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+- Users will see a great (if not huge) reduction in K8S API calls and etcd traffic
+We can increase the syncing frequency.
```
Comment (jaypipes) [inline docs/faq.md]: Should above be a bullet point?

File: docs/faq.md [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+- Advanced syncing features such as pre-post sync hooks and sync waves
+- Overall performance and efficiency improvements (registry scanning excluded) are the major gains for Flux users.
+- Given the concept GitOps is quite young, and all people involved were involved early on (or present during the birth), I think people get an even more experienced team working on core features.
```
Comment (jaypipes) [inline docs/faq.md]: s/concept GitOps/concept of GitOps/
s/I think people get/we hope this will result in/

File: docs/faq.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+----
+
+**Q**: What can current Argo users look forward to from this collaboration with Flux?
+
+**A**: We hope Argo CD might get the following:
+
+- Docker registry monitoring feature. It would be fantastic if we could extract existing Flux code into a reusable component which works for both Argo CD and Flux.
```
Comment (jaypipes) [inline docs/faq.md]: s/Flux/Flux CD/g

File: docs/faq.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+- Better cluster management experience. Right now Argo CD users use app of apps pattern which is not perfect. Perhaps we can learn from Flux community and contribute to GitOps engine to improve both Argo CD and Flux.
```
Comment (jaypipes) [inline docs/faq.md]: s/Flux/Flux CD/g

File: docs/faq.md [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+- Advanced Git related features like GPG commit verification, git secrets.
+- Simplified installations/management.
+
+----
+
+**Q**: Does this project scope just synchronization of environment (git sync operator) or does it include progressive delivery?
+
+**A**: We will be starting to work on a spec 2020.
```
Comment (jaypipes) [inline docs/faq.md]: s/spec 2020/spec for progressive delivery alignment between Argo Rollouts and Weave Flagger in 2020/

File: docs/faq.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+We also want to highlight that before we even start doing this we want to be properly introduced to each others communities, and understand each others use cases. As without this knowledge we won't be able to create something that will serve all of you.
+
+----
+
+**Q**: Was it hard to put together the design for the GitOps Engine? What was most important to you when you started putting it together?
+
+**A**: It was hard for the right reasons. We wanted to design something which ticked off all these points:
+
+1. Realistic both in theory and practice (e.g. starting a new project from scratch didn't make sense, creating a Frankenstein-like project from pieces of both projects also didn't make sense).
+1. Would allow us keep us nurturing the communities of ArgoCD and Flux, towards a common product, without jeopardizing them (e.g. without disrespecting the communities).
```
Comment (jaypipes) [inline docs/faq.md]: s/ArgoCD/Argo CD/
s/Flux/Flux CD/

**Comment by dholbach** [comment]
Thanks a lot for the review @jaypipes! :eyeglasses:

**Pull request merged**